project/MiMa.scala

```diff
@@ -534,7 +534,10 @@ object MiMa extends AutoPlugin {
       FilterAnyProblem("akka.remote.RemoteWatcher$UnwatchRemote"),
       FilterAnyProblem("akka.remote.RemoteWatcher$Rewatch"),
       FilterAnyProblem("akka.remote.RemoteWatcher$RewatchRemote"),
-      FilterAnyProblem("akka.remote.RemoteWatcher$Stats")
+      FilterAnyProblem("akka.remote.RemoteWatcher$Stats"),
+
+      // toString is available on any object, mima is confused due to a generated toString appearing #17722
+      ProblemFilters.exclude[MissingMethodProblem]("akka.japi.Pair.toString")
```

Comment:
Yup, opened a ticket already. Merging this in a sec
On Jun 23, 2015 3:56 PM, "Roland Kuhn" <notifications@github.com> wrote:

> In project/MiMa.scala
> <https://github.com/akka/akka/pull/17722#discussion_r33042982>:
>
> > @@ -534,7 +534,10 @@ object MiMa extends AutoPlugin {
> >        FilterAnyProblem("akka.remote.RemoteWatcher$UnwatchRemote"),
> >        FilterAnyProblem("akka.remote.RemoteWatcher$Rewatch"),
> >        FilterAnyProblem("akka.remote.RemoteWatcher$RewatchRemote"),
> > -      FilterAnyProblem("akka.remote.RemoteWatcher$Stats")
> > +      FilterAnyProblem("akka.remote.RemoteWatcher$Stats"),
> > +
> > +      // toString is available on any object, mima is confused due to a generated toString appearing #17722
> > +      ProblemFilters.exclude[MissingMethodProblem]("akka.japi.Pair.toString")
>
> I think this is a bug in MiMa, please report over there and move along,
> nothing to see ;-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/akka/akka/pull/17722/files#r33042982>.
>